app/femr/ui/views/medical/listVitals.scala.html

```diff
@@ -57,15 +63,44 @@
         </tr>
         <tr id="height">
             <th>Ht</th>
+            <!--- Change metric output here --->
             @for(dateIndex <- 1 to vitalMap.getDateListChronological.size) {
-                <td>@vitalMap.get("heightFeet", vitalMap.getDate(dateIndex - 1))
-                    ' @vitalMap.get("heightInches", vitalMap.getDate(dateIndex - 1))</td>
+                <td data-feet="@vitalMap.get("heightFeet", vitalMap.getDate(dateIndex - 1))"
+                    data-inches="@vitalMap.get("heightInches", vitalMap.getDate(dateIndex - 1))">
+
+                @if(viewModel.getSettings.isMetric) {
+                    @outputHeightOrNA(
+                        vitalMap.get("heightMeters", vitalMap.getDate(dateIndex - 1)),
+                        vitalMap.get("heightCm", vitalMap.getDate(dateIndex - 1)),
+                        viewModel.getSettings.isMetric,
+                        ""
+                    )
+
+                } else {
+                    @outputHeightOrNA(
+                        vitalMap.get("heightFeet", vitalMap.getDate(dateIndex - 1)),
+                        vitalMap.get("heightInches", vitalMap.getDate(dateIndex - 1)),
+                        viewModel.getSettings.isMetric,
+                        ""
+                    )
+                }
+                </td>
+            <!---  Alaa Serhan Must Format Output -->
+@*          @for(dateIndex <- 1 to vitalMap.getDateListChronological.size) {
+                    <td>@vitalMap.get("heightFeet", vitalMap.getDate(dateIndex - 1))
+                        . @vitalMap.get("heightInches", vitalMap.getDate(dateIndex - 1))</td>*@
```

Comment:
If you mean the commented out code, yes it can be removed. However, the non commented code is to format the display for metric or imperial.